packages/office-addin-sso/src/msgraph-helper.ts

```diff
@@ -3,46 +3,15 @@
     This file provides the provides functionality to get Microsoft Graph data.
 */
 
-import { ODataHelper } from "./odata-helper";
 import { showMessage } from "./message-helper";
 import * as $ from "jquery";
 
-const domain: string = "graph.microsoft.com";
-const versionURLsegment: string = "/v1.0";
-
-export async function getGraphData(
-  accessToken: string,
-  apiURLsegment: string,
-  queryParamsSegment?: string
-): Promise<any> {
-  try {
-    const oData = await ODataHelper.getData(accessToken, domain, apiURLsegment, versionURLsegment, queryParamsSegment);
-    return Promise.resolve(oData);
-  } catch (err) {
-    return Promise.reject(`Error get Graph data. \n${err}`);
-  }
-}
-
-export async function getGraphToken(bootstrapToken): Promise<any> {
-  try {
-    let response = await $.ajax({
-      type: "GET",
-      url: "/auth",
-      headers: { Authorization: "Bearer " + bootstrapToken },
-      cache: false,
-    });
-    return response;
-  } catch (err) {
-    throw new Error(`Error getting Graph token. \n${err}`);
-  }
-}
-
-export async function makeGraphApiCall(accessToken: string): Promise<any> {
+export async function makeGraphApiCall(bootstrapToken: string): Promise<any> {
   try {
     const response = await $.ajax({
       type: "GET",
       url: "/getuserdata",
-      headers: { access_token: accessToken },
+      headers: { Authorization: "Bearer " + bootstrapToken },
       cache: false,
     });
     return response;
```

Comment:
Here too I am wondering if the server is returning 400 or 401 on errors to force this try/catch to happen.

Reply:
I don't think a 400 or 401 causes an exception . . . it should just report the response to the method caller and they should handle the response.  This is how the taskpane code will handle the request for claims as well.

Reply:
cool